app/addons/documents/sidebar/SidebarControllerContainer.js

```diff
@@ -12,25 +12,102 @@
 
 import { connect } from 'react-redux';
 import SidebarComponents from './sidebar';
-import ActionTypes from './actiontypes';
+import Action from './actions';
 
-const reduxUpdatedDesignDocList = (designDocs) => {
-  return {
-    type: ActionTypes.SIDEBAR_UPDATED_DESIGN_DOCS,
-    options: {
-      designDocs: Array.isArray(designDocs) ? designDocs : []
-    }
+const getDatabase = (state) => {
```

Comment:
I think to be consistent with how we have done redux elsewhere these should go into the reducer file for sidebars.